packages/esm-patient-banner-app/src/print-identifier-sticker/print-identifier-sticker-action-button.component.tsx

```diff
@@ -10,7 +10,6 @@ interface PrintIdentifierStickerOverflowMenuItemProps {
 
 const PrintIdentifierStickerOverflowMenuItem: React.FC<PrintIdentifierStickerOverflowMenuItemProps> = ({ patient }) => {
```

Comment:
Shouldn't we delete this file, since the extension registration is also removed.